deploy-image/Dockerfile

```diff
@@ -9,29 +9,29 @@ RUN apk add --no-cache gcc musl-dev python3-dev libffi-dev openssl-dev
 
 # https://stackoverflow.com/a/57485724
 RUN apk add --update --no-cache py3-numpy py3-scipy py3-pillow
-ENV PYTHONPATH "$PYTHONPATH:/usr/lib/python3.7/site-packages"
+ENV PYTHONPATH "$PYTHONPATH:/usr/lib/python3.10/site-packages"
 
 # golang
 RUN apk add --no-cache go
 
 # aea installation
 RUN pip install --upgrade pip
-RUN pip install --upgrade --force-reinstall open-aea[all]==1.16.0
+RUN pip install --upgrade --force-reinstall open-aea[all]==1.16.0 open-aea-cli-ipfs==1.16.0
 
 # directories and aea cli config
-COPY /.aea /home/.aea
 WORKDIR /home/agents
-COPY ./packages /home/agents/packages
 RUN pip install --upgrade setuptools
+RUN aea init --reset --author default_author --remote --ipfs --ipfs-node /dns/registry.autonolas.tech/tcp/443/https
+
 # aea build script
-COPY /build.sh /build.sh
+COPY deploy-image/build.sh /build.sh
 RUN ["chmod", "+x", "/build.sh"]
 RUN [ "/build.sh" ]
 
 # optionally, specify any ports to expose here
 # EXPOSE 9000
 
 # aea entrypoint script
-COPY /entrypoint.sh /entrypoint.sh
+COPY deploy-image/entrypoint.sh /entrypoint.sh
```

Comment:
interesting
So you are building the deploy image from the root directory?

Reply:
Yeah, since the other images do the same.